tests/Auth/AuthAccessGateTest.php

```diff
@@ -299,7 +299,7 @@ public function test_after_callbacks_do_not_override_each_other()
         $gate = $this->getBasicGate();
 
         $gate->after(function ($user, $ability, $result) {
-            return $ability == 'allow' ? true : false;
+            return $ability == 'allow';
```

Comment:
Any reason why not `===` ?

Reply:
+1 for `===`

Reply:
See https://github.com/laravel/framework/pull/26139 @KennedyTedesco